daemon/api.go

```diff
@@ -62,36 +62,42 @@ var api = []*Command{
 
 var (
 	rootCmd = &Command{
-		Path: "/",
-		GET:  SyncResponse([]string{"/1.0"}).Self,
+		Path:    "/",
+		GuestOK: true,
+		GET:     SyncResponse([]string{"/1.0"}).Self,
 	}
 
 	v1Cmd = &Command{
-		Path: "/1.0",
-		GET:  v1Get,
+		Path:    "/1.0",
+		GuestOK: true,
```

Comment:
I'd call this GET-Guest-OK or something like that to be more explicit but I guess we can change that later. I do agree that POST and DELETE are always not-okay but "GuestOK" is ambiguous as to which operation is actually allowed.